src/main/webapp/utils/matches-filter.tsx

```diff
@@ -0,0 +1,41 @@
+const escapeRegExp = (string: string) => {
```

Comment:
This file was pulled over from [ddf-ui](https://github.com/codice/ddf-ui/master/ui-frontend/packages/catalog-ui-search/src/main/webapp/component/select/filterHelper.js)